src/http.ts

```diff
@@ -58,11 +57,11 @@ export class HttpClient implements IHttpClient {
     const port = parseInt(curl.port, 10);
     const headers: IHeaders = {
       'User-Agent': 'node-soap/' + version,
-      'Accept': 'text/html,application/xhtml+xml,application/xml,text/xml;q=0.9,*/*;q=0.8',
+      Accept: 'text/html,application/xhtml+xml,application/xml,text/xml;q=0.9,*/*;q=0.8',
       'Accept-Encoding': 'none',
```

Comment:
This and one below appear to be inconsistent in removal of single quotes for some fileds, but not others. Ideally, the quotes in such cases should be preserved.